docs/_posts/2019-06-19-42.2.6-release.md

```diff
@@ -0,0 +1,246 @@
+---
+title: PostgreSQL JDBC Driver 42.2.6 Released
+date: 2019-06-19 00:06:55 +0300
+categories:
+  - new_release
+version: 42.2.6
+---
+**Notable changes**
+
+### Changed
+
+### Added
+
+### Fixed
```

Comment:
This should not be here (empty).

Reply:
@jorsol what should be there ? Is this a problem with the release_notes.sh script ?

Reply:
I think we migrate to Gradle, and use Gradle-based scripting (e.g. Kotlin-based one).

Empty sections are probably caused by the fact that I executed the script twice while the first one failed due to missing Perl libraries.

Reply:
@davecramer actually the script does the right thing, maybe is a residual effect of how was executed, the flow should be edit all entries of the CHANGELOG.md without changing `[Unreleased]` tag, then the release_notes.sh does the change from Unreleased to the version of the project without `-SNAPSHOT`...

In other words, in the post file, the labels Changed, Added and Fixed are duplicate and empty.